### ISSUE TITLE
cannot reach setOnClickListener after setOnSlideCompleteListener complete suceesfully

### ISSUE THREAD

**Issue by khakimsetia** [cortinico/slidetoact#141]
after slide complete,
setOnClickListener can not function again.

```
//java8
SlideToActView slide = findViewById(R.id.Slide);

//this function not work after Slide status has Complete
slide.setOnClickListener(v -> {
   Log.d("msg", "slide onClick"); 
}
```

**Comment by cortinico** [comment]
Yes you're right. The widget was blocking click events when completed. This was fixed by #142 now

**Comment by khakimsetia** [comment]
thankyou.
how to update to use this fix version?
are new version 10 can be used?
are you have update to gradle dependencies.
